scripts/lint/re-frame-in-quo-components.sh

```diff
@@ -1,10 +1,28 @@
 #!/usr/bin/env sh
 
-INVALID_CHANGES=$(grep -E -r '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.|status-im2\.)' --include '*.cljs' --include '*.clj' './src/quo')
+INVALID_CHANGES=$(grep -E -r '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.)' --include '*.cljs' --include '*.clj' './src/quo')
 
 if test -n "$INVALID_CHANGES"; then
     echo "WARNING: re-frame, status-im are not allowed in quo components"
     echo ''
     echo "$INVALID_CHANGES"
     exit 1
 fi
+
+INVALID_CHANGES2=$(grep -E -r '(status-im\.)' --include '*.cljs' --include '*.clj' './src/utils')
+
+if test -n "$INVALID_CHANGES2"; then
+    echo "WARNING: status-im are not allowed in utils package"
+    echo ''
+    echo "$INVALID_CHANGES2"
+    exit 1
+fi
+
+INVALID_CHANGES3=$(grep -E -r '(status-im\.)' --include '*.cljs' --include '*.clj' './src/react_native')
+
+if test -n "$INVALID_CHANGES3"; then
+    echo "WARNING: status-im are not allowed in react-native package"
+    echo ''
+    echo "$INVALID_CHANGES3"
+    exit 1
+fi
```

Comment:
A better way would be : 
```bash

check_invalid_changes() {
    local pattern=$1
    local path=$2
    local message=$3

    local invalid_changes=$(grep -E -r "$pattern" --include '*.cljs' --include '*.clj' "$path")

    if test -n "$invalid_changes"; then
        echo "WARNING: $message"
        echo ''
        echo "$invalid_changes"
        exit 1
    fi
}


check_invalid_changes '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.)' './src/quo' 're-frame, status-im are not allowed in quo components'
check_invalid_changes '(status-im\.)' './src/utils' 'status-im are not allowed in utils package'
check_invalid_changes '(status-im\.)' './src/react_native' 'status-im are not allowed in react-native package'
```

Reply:
probably :) I'd leave it to you

Reply:
Thanks, I'll improve this in a follow up :)